racetrack_client/racetrack_client/manifest/load.py

```diff
@@ -40,12 +40,12 @@ def parse_manifest_or_empty(manifest_yaml: Optional[str]) -> Optional[Manifest]:
     try:
         return parse_yaml_datamodel(manifest_yaml, Manifest)
     except Exception as e:
-        logger.error(f'Fatman Manifest YAML contains syntax error: {e}')
+        logger.error(f'Job Manifest YAML contains syntax error: {e}')
         return None
 
 
 def get_manifest_path(workdir_or_file: str) -> Path:
     manifest_path = Path(workdir_or_file)
     if manifest_path.is_dir():
-        manifest_path = manifest_path / FATMAN_MANIFEST_FILENAME
+        manifest_path = manifest_path / JOB_MANIFEST_FILENAME
```

Comment:
We should keep it backwards compatible with old `fatman.yaml` if we want to have it in Racetrack v2, not v3.

I suggest to do something like this:
```python
FORMER_MANIFEST_FILENAME = 'fatman.yaml'


def get_manifest_path(workdir_or_file: str) -> Path:
    if Path(workdir_or_file).is_dir():
        manifest_path = Path(workdir_or_file) / FATMAN_MANIFEST_FILENAME
        if manifest_path.is_file():
            manifest_path

        former_path = Path(workdir_or_file) / FORMER_MANIFEST_FILENAME
        if former_path.is_file():
            logger.warning(f'You are using old manifest file name "{FORMER_MANIFEST_FILENAME}". Please rename it to "{FATMAN_MANIFEST_FILENAME}"')
            former_path
        
        raise FileNotFoundError(f"local manifest file '{manifest_path}' doesn't exist")

    else:
        manifest_path = Path(workdir_or_file)
        if not manifest_path.is_file():
            raise FileNotFoundError(f"local manifest file '{manifest_path}' doesn't exist")

        if manifest_path.name == FORMER_MANIFEST_FILENAME:
            logger.warning(f'You are using old manifest file name "{FORMER_MANIFEST_FILENAME}". Please rename it to "{FATMAN_MANIFEST_FILENAME}"')
            
        return manifest_path

```